app/models/product_import/reset_absent.rb

```diff
@@ -0,0 +1,33 @@
+module ProductImport
+  class ResetAbsent
```

Comment:
I prefer classes with nouns as names and more explicit, in this case I believe AbsentEnterprisesResetter
And I'd make the main method more contextual, in this case, reset() instead of call()

Reply:
yes, sounds good. I am learning Ruby conventions. Thanks!
I must say I dislike the conventions. It's OO, these are objects, not actions.
Services should have single responsibility, but that's very different from having a single method in its API.
For example:
- PasswordManager service updates the password but can also reset it.
- UserBuilder can create user in slightly different ways.

Reply:
Sure! it's just that services following these conventions in Rails are simply implementing the Command pattern. We can totally differ from these and have more complex services and still store them in `app/services`. However, if we think the service is simple enough to have just `#call`, `#perform` or whatever method, is worth sticking to these conventions.